web-marketplace/src/hooks/useQueryIsProjectAdmin.ts

```diff
@@ -4,20 +4,24 @@ import { useLedger } from 'ledger';
 import { getProjectsByAdminQuery } from 'lib/queries/react-query/ecocredit/getProjectsByAdmin/getProjectsByAdmin';
 import { useWallet } from 'lib/wallet/wallet';
 
-export function useQueryIsProjectAdmin(): boolean {
+type Props = {
+  address?: string;
+};
+
+export function useQueryIsProjectAdmin({ address }: Props) {
   const { ecocreditClient } = useLedger();
   const { wallet } = useWallet();
-  const address = wallet?.address;
+  const currentAddress = wallet?.address;
 
-  const { data: projectsByAdmin } = useQuery(
+  const { data: projectsByAdmin, isFetching } = useQuery(
     getProjectsByAdminQuery({
       enabled: !!address && !!ecocreditClient,
       client: ecocreditClient,
-      request: { admin: address },
+      request: { admin: address ?? currentAddress },
```

Comment:
can we use the same naming as in the other hooks with 
```
const walletAddress = wallet?.address;
const activeAddress = address ?? walletAddress;
```

we could also extract this into a reusable function

Reply:
Don't think an util is worth it here but I will align the naming.